react/features/toolbox/components/Toolbox.native.js

```diff
@@ -21,7 +21,9 @@ import {
     abstractMapDispatchToProps,
     abstractMapStateToProps
 } from '../functions';
+
 import styles from './styles';
+import AudioRouteButton from './AudioRouteButton';
```

Comment:
I don't think we sort in a case-sensitive manner so `AudioRouteButton` should be before `styles`.